src/main/scala/io/qbeast/core/model/DataWriter.scala

```diff
@@ -63,7 +63,7 @@ trait DataWriter[DATA, DataSchema, FileDescriptor] {
    * @return
    *   the sequence of files written and deleted
    */
-  def compact(
+  def optimize(
```

Comment:
Please also update the method description

Reply:
For example, "Reindex the selected files to bring the data layout closer to the dictates of the index."

Reply:
Do we have documentation explaining what optimization means now?

Reply:
To be found in `/docs/QbeastFormat.md`